pyFAST/fastfarm/FASTFarmCaseCreation.py

```diff
@@ -537,22 +483,37 @@ def _create_dir_structure(self):
                 for seed in range(self.nSeeds):
                     seedPath = os.path.join(casePath, f'Seed_{seed}')
                     if not os.path.exists(seedPath):  os.makedirs(seedPath)
-                    turbsimPath = os.path.join(seedPath, 'TurbSim')
-                    if not os.path.exists(turbsimPath):  os.makedirs(turbsimPath)
+                    if self.inflowStr == 'MannBox':
+                        path=os.path.dirname(self.path)         
+                        WindPath = os.path.join(path, 'Wind')
```

Comment:
I'm not sure I'm following your naming convention. Can you explain better what you are trying to do here and how it would fit on workflows already in place using the updated class?